emojivoto-web/web/web.go

```diff
@@ -374,6 +375,12 @@ func writeError(err error, w http.ResponseWriter, r *http.Request, status int) {
 	json.NewEncoder(w).Encode(errorMessage)
 }
 
+func handle(path string, h func (w http.ResponseWriter, r *http.Request)) {
```

Comment:
`web` needs this because the gRPC stuff comes OOB?

Reply:
It's a consequence of how we've structured the web routing.  We have to install a server filter but because emojivoto-web registers all of its http handlers individually, we have to put the server filter on each one.  Using some kind of router would simplify this.